src/rust/engine/process_execution/src/lib.rs

```diff
@@ -87,6 +87,8 @@ pub enum Platform {
   Macos_x86_64,
   Macos_arm64,
   Linux_x86_64,
+  Linux_arm64,
+  Linux_aarch64,
```

Comment:
These are synonyms, right? I'm thinking we only want one entry in the enum. Specifically `arm64`, which is what we use for macOS already